src/client/components/Header/Header.component.js

```diff
@@ -17,13 +22,18 @@ export default function Header({ isAuthenticated, username, link }) {
       {/* ICONS */}
       <div className="icons-right">
         <img className="icons" src={faShoppingCart} alt="shopping-cart" />
-        <img className="icons" src={faHeart} alt="favorite" />
+        <Link to='favorites'><button type='button'><img className="icons" src={faHeart} alt="favorite" /></button></Link>
         <img className="icons" src={faUser} alt="login" />
 
         {isAuthenticated ? (
-          <div className="login icons"> Hello {username}</div>
+          <>
+            <button onClick={() => signOut()} type="button">
+              Logout
+            </button>
+            <div className="login icons"> Hello {user.email}</div>
+          </>
```

Comment:
This is working 👍 but it needs a little style :)

Reply:
Styling not yet done. Please test for functionality for now.